Makefile

```diff
@@ -66,14 +66,14 @@ manifests: controller-gen ## Generate CRD manifests
 generate: controller-gen ## Generate CRD code
 	$(CONTROLLER_GEN) object:headerFile="build/boilerplate.go.txt" paths="./pkg/apis/..."
 
-docker-build: manifests generate test ## Build the docker image
-	docker build . -t $(IMG) --load
+docker-build: manifests generate ## Build the docker image
+	docker build . -t $(IMG)
 
 crds-to-chart: ## copy crds to helm chart directory
-	cp deploy/crds/*.yaml $(HELM_DIR)/templates/crds/; \
-	for i in deploy/charts/secret-manager/templates/crds/*.yaml; do \
+	cp deploy/crds/*.yaml $(HELM_DIR)/crds/; \
```

Comment:
This was a specific decision to not use the Helm crd directory because the crd directory does not handle crd upgrades. If crd's are instead in the templates directory then they are upgraded. 

Our intention is that all CRD upgrades will follow semantic version, so this should be safe to upgrade when the chart upgrades. If Helm changes its crd upgrade implementation then we could reconsider moving to the CRD directory.

Reply:
I see, that make sense :+1: . Can we separate the CRDs into it's own Chart? Installing CRDs from the template dir will result either in a conflict or a a tedious e2e test setup.

Reply:
For now it would probably be good enough to:
1. `kubectl apply -f ./deploy/crds/` 
2. use helm with `installCRD: false`

what do you think?

Reply:
Yeah that is why crd's are in their own director. I believe applying the directory and installing with installCRD's false is currently possible. Does that currently have issues?

I believe we were later going to setup a release artifact of the crd's on their own as well.

Reply:
> Does that currently have issues?

nope, that works as intended!